modules/hcp-configuring-oauth-console.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-authentication-authorization.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-configuring-oauth-console_{context}"]
+= Configuring the OAuth server for a hosted cluster by using the web console
+
+You can configure the internal OAuth server for your hosted cluster by using the {product-title} web console. You can configure OAuth for the supported identity providers such as `oidc`, `htpasswd`, `keystone`, `ldap`, `basic-authentication`, `request-header`, `github`, `gitlab`, and `google`. Adding any identity provider in the OAuth configuration removes the default `kubeadmin` user provider.
```

Comment:
"supported identity providers such as" implies that this is an incomplete list, but the length of the list (9 providers) makes me wonder if this is a complete list? 

If it is meant to be only a selection of options from a larger list, consider just picking a few to highlight.

If it is a complete list, consider something like:
```suggestion
You can configure the internal OAuth server for your hosted cluster by using the {product-title} web console. 

You can configure OAuth for the following supported identity providers:
* `oidc`
* `htpasswd`
* `keystone`
* `ldap`
* `basic-authentication`
* `request-header`
* `github`
* `gitlab`
* `google`

Adding any identity provider in the OAuth configuration removes the default `kubeadmin` user provider.
```

Edit: I see this is taken from the CLI version, so use your best judgement here (if you want to tackle this for both later, that's fine imo)